.travis.yml

```diff
@@ -256,6 +256,10 @@ matrix:
     - php: 7.1
       env: DB=pgsql POSTGRESQL_VERSION=9.6
 
+before_install:
+  # Force hhvm PHP 7 mode
+  - if [[ $TRAVIS_PHP_VERSION = hhv* ]]; then echo hhvm.php7.all=1 >> /etc/hhvm/php.ini; fi
```

Comment:
? This were just two examples to show off that the pattern `hhvm*` matches `hhvm` as well as `hhvm-3.12`. Nevermind, I approved the PR already.